libs/ledgerjs/packages/hw-app-exchange/src/Exchange.ts

```diff
@@ -31,6 +31,8 @@ const PROCESS_TRANSACTION_RESPONSE = 0x06;
 const CHECK_TRANSACTION_SIGNATURE = 0x07;
 const CHECK_PAYOUT_ADDRESS = 0x08;
 const CHECK_REFUND_ADDRESS = 0x09;
+// const CHECK_REFUND_ADDRESS_AND_DISPLAY = 0x09;
+// const CHECK_REFUND_ADDRESS_NO_DISPLAY = 0x0c;
```

Comment:
Should it be remove ?

Reply:
Not yet as they will be used in future commit. This are new APDUs to use with Exchange flow.